pkg/filters/cgroup-bpf/bpf.go

```diff
@@ -0,0 +1,118 @@
+package bpf
```

Comment:
Should this be called `package cgroupbpf` to differentiate cgroupbpf and tcbpf?

Reply:
Ouch, yes. It should be fixed now.